lib/pxconfig.js

```diff
@@ -268,7 +265,7 @@ function pxDefaultConfig(PX_INTERNAL) {
         SENSITIVE_HEADERS: ['cookie', 'cookies'],
         PROXY_URL: '',
         SEND_PAGE_ACTIVITIES: true,
-        DEBUG_MODE: false,
+        DEBUG_MODE: LoggerSeverity.FATAL,
```

Comment:
when we changed it from `true` `false` to loggerSeverity `debug` `error` maybe the config name should also be changed? it is no longer debug mode but more like logger mode or logger severity.
what do you think?

Reply:
Yes, good point.